Documentation/DIContainer.md

```diff
@@ -1,18 +1,21 @@
 # DI Container
 
-Dependency Injection (DI) Container manages dependencies of your system. First you register the way how the dependencies or actual types should be resolved. Then you use the DI container to get instances whose dependencies or actual types are resolved by the DI container. In Swinject, `Container` class represents the DI container.
+Dependency Injection (DI) Container manages the type dependencies of your system. First, you register the  types that should be resolved, with their dependencies. Then you use the DI container to get instances of those types whose dependencies are automatically resolved by the DI container. In Swinject, the `Container` class represents the DI container.
 
-In the field of Dependency Injection or Inversion of Control, different terms are used or terms are used differently. Swinject uses the following terms.
+The terms Dependency Injection and Inversion of Control are synonymous. We use the former term, or DI for short.  Confusingly, different DI frameworks use different terms to describe the core concepts. Swinject uses the following terms:
```

Comment:
I rewrote this section again, using the phrase above as well as adding some links to Wikipedia for DI and IoC.